lib/msf/core/payload/php.rb

```diff
@@ -70,38 +69,38 @@ def php_system_block(options = {})
     in_array    = '$' + Rex::Text.rand_text_alpha(rand(4) + 4)
 
     setup = "
-      if (FALSE !== strpos(strtolower(PHP_OS), 'win' )) {
+      if (FALSE !== stristr(PHP_OS, 'win' )) {
         #{cmd}=#{cmd}.\" 2>&1\\n\";
       }
       #{is_callable}='is_callable';
       #{in_array}='in_array';
       "
     shell_exec = "
-      if(#{is_callable}('shell_exec')and!#{in_array}('shell_exec',#{dis})){
+      if(#{is_callable}('shell_exec')&&!#{in_array}('shell_exec',#{dis})){
         #{output}=shell_exec(#{cmd});
       }else"
     passthru = "
-      if(#{is_callable}('passthru')and!#{in_array}('passthru',#{dis})){
+      if(#{is_callable}('passthru')&&!#{in_array}('passthru',#{dis})){
         ob_start();
-        passthru(#{cmd});
+        `#{cmd}`;
```

Comment:
This should stay, per the linked php doc using backtic would end up calling `shell_exec` which was what the test above was looking for.

```suggestion
        passthru(#{cmd});
```